spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/SpatialBatchNormalization.scala

```diff
@@ -303,9 +305,6 @@ class SpatialBatchNormalization(
   }
 
   override def zeroGradParameters(): Unit = {
```

Comment:
maybe we can just remove the method. I remember the default impl is do nothing, right?